bindings/kitex/kitex_output.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kitex
+
+import (
+	"context"
+	"sync"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/kit/logger"
+)
+
+type kitexOutputBinding struct {
+	ctxCache  map[string]*kitexContext
+	cacheLock sync.RWMutex
+	logger    logger.Logger
+}
+
+var kitexBinding *kitexOutputBinding
```

Comment:
We should not be using singletons because:

1. We can have multiple components initialized of the same type
2. This is not safe for concurrent access
3. We should be able to Close components

Reply:
Got it，I've learned it.